### PR TITLE
Fix #2056 create viewModel for speakerDetails activity

### LINKED ISSUE #2056: Create ViewModel for SpeakerDetailsActivity
**Actual Behaviour**

There is no ViewModel for the SpeakerDetailsActivity

**Expected Behaviour**

There should be one.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by nikit19**
Fixes #2056 

Changes:Created ViewModel for the SpeakerDetailsActivity

Screenshots for the change: 
screenshot_20171231-123503

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/activities/SpeakerDetailsActivity.java [hunk 1]
```diff
@@ -288,18 +290,15 @@ protected void onResume() {
...
+        speakerDetailsViewModel.getSpeaker(speakerName).observe(this, speakerData -> {
+            selectedSpeaker = speakerData;
             loadSpeakerDetails();
         });
     }
 
     @Override
     protected void onStop() {
         super.onStop();
-        if(speaker != null)
```
Comment (dr0pdb) [inline android/app/src/main/java/org/fossasia/openevent/activities/SpeakerDetailsActivity.java]: This method is redundant.

**Review by dr0pdb — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/org/fossasia/openevent/activities/SpeakerDetailsActivity.java (hunk 1 above)
Comment (nikit19) [reply android/app/src/main/java/org/fossasia/openevent/activities/SpeakerDetailsActivity.java]: :+1:

**Comment by dr0pdb** [comment]
@nikit19 While you're at it, I think you can also delete the redundant ```onSavedInstanceState``` method [here](https://github.com/nikit19/open-event-android/blob/cea1be60a1ed4af216815c872085d52f8f611ecc/android/app/src/main/java/org/fossasia/openevent/activities/SpeakerDetailsActivity.java#L311) which does nothing except calling its super.

**Comment by iamareebjamal** [comment]
@nikit19 Please change PR to use RealmDataWrapper instead

**Comment by nikit19** [comment]
@iamareebjamal updated the PR to use RealmDataWrapper.
Please review,
Thank you

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**